mainapp/templates/account/signup.html

```diff
@@ -7,6 +7,8 @@
 {% block head_title %}{% trans "Signup" %}{% endblock %}
 
 {% block content %}
+<head>
```

Comment:
`<head>` is added. Is it an intended change?

Reply:
Removed